Gemfile

```diff
@@ -49,6 +49,7 @@ gem 'bootsnap', require: false
 # gem "image_processing", "~> 1.2"
 
 group :development, :test do
+  gem 'rspec-rails'
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
```

Comment:
[OPTIONAL] 🚀 It seems that the LICENSE file has not been properly linked in the README. When I tried to open the LICENSE link in the README, it led me to a 💥broken link💥. I understand that things like this can happen and I believe it was an oversight on your part. No worries, it happens to the best of us 🤣. I kindly ask you to connect the LICENSE file to the README so it can be easily accessed by others. If you need any assistance or have any questions, please let me know 🤗.